packages/react-cosmos-router-proxy/src/LocationInterceptor.js

```diff
@@ -0,0 +1,22 @@
+import { Component } from 'react';
+import { object } from 'prop-types';
+
+// Heads up: This is tied to React Router internals and is subject to change
+const getPathFromContext = context => context.router.route.location.pathname;
```

Comment:
You're right. Addressed this in https://github.com/react-cosmos/react-cosmos/pull/406/commits/23445faf93fc3b305f2fc40185073adb27ebd33b

I hesitated to use `withRouter` first because it adds another `<Route>` element, and I wanted to test that the Proxy renders exactly 1 or 0 Route elements, based on scenario. But I changed the assumptions and look for a Route with a specific prop.